tests/console/validate_zfcp_multipath.pm

```diff
@@ -37,7 +37,7 @@ sub verify_multipath_block_devices {
     my $mpath = script_output "multipath -l";
     my @mpath = split /\n/, $mpath;
     # check for 50GB disk (multiline match)
-    assert_matches qr/^3600507638081855cd800000000000076.+\n^size=50G/m, $mpath, "Cannot find 50GB disk on WWID *076";
+    assert_matches qr/size=50G/m, $mpath, "Cannot find 50GB disk";
```

Comment:
according to https://suse.slack.com/archives/C02CLB2LB7Z/p1710339859125099?thread_ts=1710329927.799739&cid=C02CLB2LB7Z we should extract the wwpn and add it here to vaidate, not remove it.